File: src/1-Problem01.py
"""
    NIM/Nama  : 13516002/Antonio Setya
    Topik     : Percabangan
    Deskripsi : Memunculkan jurusan dari seorang mahasiswa jika diberikan NIM-nya
"""

# Input
nama = input("Masukan Nama : ") # Menyimpan nama
nim = input("Masukan NIM : ") # Menyimpan NIM
jurusan = "???" # Menyimpan jurusan. Default adalah ???, bilamana program tidak dapat menentukan jurusannya

# Memroses NIM. Ini aman karena NIM selalu dipastikan berupa 8 karakter/digit
if (nim[0] == "1"):
    if (nim[1] == "3"):
        if (nim[2] == "5"):
            jurusan = "Teknik Informatika"
    elif (nim[1] == "8"):
        if (nim[2] == "0"):
            jurusan = "Teknik Tenaga Listrik"
        elif (nim[2] == "1"):
            jurusan = "Teknik Telekomunikasi"
        elif (nim[2] == "2"):
            jurusan = "Sistem dan Teknologi Informasi"

# Output
print(nama + " adalah mahasiswa " + jurusan)

File: src/1-Problem01.cpp
#include <iostream>
#include <cstring>

using namespace std;
int main(){
	//Deklarasi Variabel
	string nama, nim;
	string jurusan;
	
	//Algoritma
	cout<< "Masukan Nama : ";
	cin>> nama ;
	cout<< "Masukan NIM : ";
	cin>> nim; //masukan nim diasumsikan selalu 8 digit
	
	jurusan ="???";
	if (nim[0]=='1') {
		if (nim[1]=='3'){
			if (nim[2]=='5')
				jurusan="Teknik Informatika";}
		else if (nim[1] =='8'){
			if (nim[2]=='0')
				jurusan = "Teknik Tenaga Listrik";
			else if (nim[2]=='1')
				jurusan = "Teknik Telekomunikasi";
			else if (nim[2]=='2')
				jurusan = "Sistem Teknologi dan Informasi";
			}
		}
	
	cout<< nama << " adalah mahasiswa "<< jurusan <<endl;

  return 0;
 }
